net-misc/minidlna/minidlna-1.3.3.recipe

```diff
@@ -46,14 +46,11 @@ REQUIRES="
 	lib:libz$secondaryArchSuffix
 	"
 
-# gettext by package name instead of cmd:gettext, because those commands have "compat > 0".
```

Comment:
@OscarL could you check this? Should be covered by the cmd's in BUILD_PREREQUIRES.